tests/PathOrderMonotonicTest.cpp

```diff
@@ -0,0 +1,286 @@
+//Copyright (c) 2021 Ultimaker B.V.
+//CuraEngine is released under the terms of the AGPLv3 or higher.
+
+#include <string>
+
+#include <gtest/gtest.h>
+#include <clipper.hpp>
+
+#include "../src/infill.h"
+#include "../src/utils/linearAlg2D.h"
+#include "../src/utils/math.h"
+#include "../src/PathOrderMonotonic.h"
+#include "../src/utils/polygon.h"
+#include "ReadTestPolygons.h"
+
+#define TEST_PATHS_SVG_OUTPUT
```

Comment:
Do we want to leave this on during CI?

Reply:
Definitely not! Must've accidentally left this in while testing.
Done here: cf051f0d8e32f2b1ff34d5a74aa32ecbab0a80bf